### DISCUSSION TITLE
最新版本进不去港澳台番剧播放页面

### DISCUSSION THREAD

**Discussion by YFTX2015** [ywmoyue/biliuwp-lite#848] [bug]
### Bug 描述

不管是挂cfw还是v2rayn，进app能正常显示更新条目，但港澳台番剧点进去就出错。

### 操作步骤

```plain text
uwp回环限制已解除.先不挂梯子，设置里面替换cdn链接包括下面的服务器选择每项都试过，代理设置里面默认设置以及公共服务器也都试过。按分区——番剧——连载动画能找到最新更新的港澳台番剧，但点进去就出错。然后挂梯子，cfw跟v2rayn都试了，替换cdn链接以及代理设置也每项都试过，点击港澳台番剧名仍然报错，无法显示播放界面。
```

### 期望结果

建议像22与33的秘密基地那样增加一个海外视频加速的功能开关。至少在本人系统环境下，隔壁app在开启这个功能后，港澳台番剧播放、下载都很少出问题，而且设置起来也很方便。这边的代理设置太复杂了。

### 截图

![屏幕截图(1)](https://github.com/user-attachments/assets/b1f3321f-c83d-4655-9509-366bdf3f24e5)
![屏幕截图(3)](https://github.com/user-attachments/assets/376340c3-e293-4c34-ac17-dbbfa5376f4b)

### 日志

[20241003.log](https://github.com/user-attachments/files/17247371/20241003.log)
[20241004.log](https://github.com/user-attachments/files/17247372/20241004.log)

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 11 专业工作站版 Insider Preview Dev 27718.1000

### 设备类型

其他

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
复现了，这个页面应该有bug，这个页面我没用过也没人反馈过所以没处理，下个版本看看能不能处理下

要看港澳台番剧可以在哔哩哔哩番剧出差的up个人页面搜索并点击进去，也可以直接搜索后设置地区并切换到番剧Tab点击搜索到的番剧点进去

关于代理的设置我感觉是比较简单的，处理逻辑是先直接请求，请求报错后再使用设置好的代理服务器去请求，用户其实不需要关心要不要开关代理

**Comment by YFTX2015** [comment]
我习惯用那个顺序找新番，由于界面有bug，导致我认为代理也有问题。那么我再试试搜索看看代理会不会出问题。

**Comment by ywmoyue** [comment]
4.7.6+ 已处理
